drivers/hmis_external_apis/app/models/concerns/hmis_external_apis/externally_identified_mixin.rb

```diff
@@ -0,0 +1,25 @@
+###
+# Copyright 2016 - 2023 Green River Data Analysis, LLC
+#
+# License detail: https://github.com/greenriver/hmis-warehouse/blob/production/LICENSE.md
+###
+
+# common behavior for referral processing
+module HmisExternalApis
+  module ExternallyIdentifiedMixin
+    extend ActiveSupport::Concern
+    class_methods do
+      def first_by_external_id(cred:, id:)
+        id_scope = HmisExternalApis::ExternalId
+          .where({ value: id, remote_credential: cred, source_type: name })
```

Comment:
Is this a special case for where?  I've not seen passing an explicit hash before.

Reply:
Thanks for catching that. They are equivalent afaik. I must have added enclosing braces accidentally. We should prefer using keyword args over a hash just to keep the style consistent